packages/paypal-commerce-integration/src/components/PayPalCommercePaymentMethodComponent.tsx

```diff
@@ -1,24 +1,32 @@
-import React, { FunctionComponent, useEffect } from 'react';
-
 import {
     PayPalCommerceAlternativeMethodsPaymentOptions,
-    PayPalCommercePaymentInitializeOptions,
     PayPalCommerceCreditPaymentInitializeOptions,
+    PayPalCommercePaymentInitializeOptions,
     PayPalCommerceVenmoPaymentInitializeOptions,
 } from '@bigcommerce/checkout-sdk';
+import React, { FunctionComponent, useEffect, useRef } from 'react';
+
 import { PaymentMethodProps } from '@bigcommerce/checkout/payment-integration-api';
 
-type PayPalCommerceProvidersPaymentInitializeOptions = PayPalCommerceAlternativeMethodsPaymentOptions
-    & PayPalCommerceCreditPaymentInitializeOptions
-    & PayPalCommercePaymentInitializeOptions
-    & PayPalCommerceVenmoPaymentInitializeOptions;
+type PayPalCommerceProvidersPaymentInitializeOptions =
+    PayPalCommerceAlternativeMethodsPaymentOptions &
+        PayPalCommerceCreditPaymentInitializeOptions &
+        PayPalCommercePaymentInitializeOptions &
+        PayPalCommerceVenmoPaymentInitializeOptions;
```

Comment:
Here might be eslint warning with tabs. May I ask you to check and fix that?

Reply:
I ran the `eslint` command for this file and there are no problems with `eslint` errors or warnings. I added the scope for this package because there was not any in the `package.json`. Thanks!